ce/tests/conftest.py

```diff
@@ -231,14 +231,19 @@ def make_data_file_variable(file, var_name=None, grid=grid_anuspline):
         var_name_to_alias = {
             'tasmin': tasmin,
             'tasmax': tasmax,
-        }
+        }[var_name]
+        variable_cell_methods = {
```

Comment:
Is it worth adding a test here that has a cell_method "mean over days" to ensure that the code works into the future when we properly attribute our normal cases?

Reply:
Yes, that would be a good idea!